install_vendored_dynamic_framework/after/Pods/Target Support Files/Pods-SampleApp/Pods-SampleApp-frameworks.sh

```diff
@@ -99,13 +99,11 @@ strip_invalid_archs() {
 
 if [[ "$CONFIGURATION" == "Debug" ]]; then
   install_framework "${PODS_ROOT}/Realm/realm-swift-0.94.1/ios/swift-1.2/Realm.framework"
-  install_dsym "${PODS_ROOT}/Realm/realm-swift-0.94.1/ios/swift-1.2/Realm.framework.dSYM"
-  install_framework "$BUILT_PRODUCTS_DIR/RealmSwift/RealmSwift.framework"
+  install_framework "${BUILT_PRODUCTS_DIR}/RealmSwift/RealmSwift.framework"
 fi
 if [[ "$CONFIGURATION" == "Release" ]]; then
   install_framework "${PODS_ROOT}/Realm/realm-swift-0.94.1/ios/swift-1.2/Realm.framework"
-  install_dsym "${PODS_ROOT}/Realm/realm-swift-0.94.1/ios/swift-1.2/Realm.framework.dSYM"
```

Comment:
This is correct. The method logic now checks if the dSYM actually exists before adding it to the script blindly. The shell script also protects for errors of files being added that are missing and will not fail.